engine/tester/config_parse.c

```diff
@@ -2713,8 +2733,11 @@ get_test_package(xmlNodePtr root, tester_cfg *cfg,
 
     node = root->children;
 
-    /* Get mandatory description */
-    if (node != NULL &&
+    if (pkg->objective != NULL)
+    {
+        node = xmlNodeNext(node);
```

Comment:
You must check that node is not NULL before calling xmlNodeNext() since the function asserts it.